src/현정빈/Myclass.java

```diff
@@ -0,0 +1,148 @@
+package 현정빈;
+
+import jun.ManagementSystem;
+import jun.Member;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Myclass implements ManagementSystem {
+    HashMap<Integer,Member> map = new HashMap<>(); //컬렉션 중 Map을 선택하여 HashMap 사용
+    // 키는 integer 타입, 값은 Member 타입인 키-값쌍 구조의 map 생성
+
+    /**
+     * 저장소에 멤버를 추가하는 메서드,
+     * 이미 존재하는 id라면 추가에 실패해야 함,
+     * 추가가 성공하면 true를 반환
+     */
+    @Override
+    public boolean addMember(String name, int age, int id) {
+        Member newMember = new Member(name, age, id);
+        if(!map.containsKey(id)) {//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 false라면
+            map.put(id, newMember); //map에 지정된 키(id)와 값(객체 newMember)을 저장
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
+    /**
+     * 저장소에서 멤버를 삭제하는 메서드,
+     * 존재하지 않는 id라면 false를 반환
+     */
+    @Override
+    public boolean removeMember(int id) {
+        if(!map.containsKey(id)){//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 false라면
+            return false;
+        }
+        else {
+            map.remove(id); //지정된 id와 연결된 값을 map에서 제거
+            return true;
+        }
+    }
+
+    /**
+     * 멤버를 수정하는 메서드 1,
+     * id를 기준으로 멤버를 찾아서, 이름을 수정함,
+     * 존재하지 않는 id라면 false를 반환
+     */
+    @Override
+    public boolean updateMember(int id, String name) {
+        if(map.containsKey(id)) {//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 true라면
+            map.get(id).setName(name);//map에서 지정된 id에 연결된 값(참조값=주소)을 반환하면서 Member의 메소드를 사용할 수 있게 된다
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```

Comment:
동균 코어님이 말씀하신대로 else 예약어를 지양하시는게 좋지만
불가피하게 작성하시게 된다면 IDE가 추천하는 방식대로 작성하시는게 좋습니다
굳이 불필요한 공백을 만들 필요는 없거든요!
```suggestion
    @Override
    public boolean updateMember(int id, String name) {
        if(map.containsKey(id)) {//지정된 id를 포함하고 있는지 여부를 반환, 반환값이 true라면
            map.get(id).setName(name);//map에서 지정된 id에 연결된 값(참조값=주소)을 반환하면서 Member의 메소드를 사용할 수 있게 된다
            return true;
        } else {
            return false;
        }
    }
```